.evergreen/atlas_data_lake/drivers-test-local.yaml

```diff
@@ -0,0 +1,66 @@
+environment: development
```

Comment:
Can this file use `.yml` as its extension for consistency with https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/ocsp/certs.yml and most of our spec tests?

Reply:
no restriction here from ADL.